app/router/group.py

```diff
@@ -7,8 +7,8 @@
 group_db_url = settings.db_url + "/group/"
 
 
-@router.get("/{group_id}", response_model=GroupResponse)
-def get_group_data(group_id: str):
+@router.get("/", response_model=GroupResponse)
+def get_group_data(request: Request):
     """
     This API returns group details corresponding to the provided group ID, if the ID exists in the database
```

Comment:
Looks like the URL schema has changed? Maybe update the docs?

Reply:
Aren't the docs automatically be generated?